src/api.c

```diff
@@ -731,3 +731,28 @@ API int seccomp_export_bpf(const scmp_filter_ctx ctx, int fd)
 
 	return 0;
 }
+
+/* NOTE - function header comment in include/seccomp.h */
+API int seccomp_export_bpf_mem(const scmp_filter_ctx ctx, void *buf,
```

Comment:
Would `seccomp_export_bpf_buf()` be a better name?  I don't have any strong feelings here...

Reply:
i don't have a pref either way.  i'll just use whatever gets approved ;).